pkgs/development/tools/analysis/qcachegrind/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, bash, cmake,
+  qt, graphviz,
+  qmakeHook, makeQtWrapper,
+  perl, python, php
```

Comment:
According to the [style guide](https://nixos.org/nixpkgs/manual/#sec-syntax) (“Function formal arguments”), lines with arguments do not end with commas.